scripts/jestSSRSetup.js

```diff
@@ -46,6 +46,8 @@ Context.within(new ArtificialServerContext(), () => {
   // Emotion detects whether the document exists at module load time >:(
   // so we have to preload it here so that it doesn't load in a non-server context
   require('@emotion/cache');
+
+  require('use-ssr')
```

Comment:
Do you mind leaving a comment that explains what's going on here and why we're requiring it

Reply:
I might just change the comment above to explain both 😄

Reply:
Done